azure-pipelines.yml

```diff
@@ -24,6 +24,54 @@ variables:
     openblas_version: 0.3.9
 
 jobs:
+- job: pre_release_deps_source_dist
+  condition: and(succeeded(), ne(variables['Build.SourceBranch'], 'refs/heads/master'))  # skip for PR merges
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.7'
+      addToPath: true
+      architecture: 'x64'
+  - script: >-
+      python -m pip install --upgrade
+      pip==20.2.4
```

Comment:
What's going on here? `pip` is downgraded to 20.2.4 after being upgraded on the line above it looks like, is that necessary?

Reply:
Or wait, is this all one command? If so, why not put it on a single line?

Reply:
The pip version pin is present in the Travis yml as far as I can tell; separate alphabetically-sorted lines for deps is just for readability, similar to apt-get package specifiers in Dockerfiles; we can condense to a single long line--I don't mind..